src/scout_apm/pyramid.py

```diff
@@ -24,34 +23,42 @@ def includeme(config):
 
 def instruments(handler, registry):
     def scout_tween(request):
-        try:
-            tr = TrackedRequest.instance()
-            span = tr.start_span(operation="Controller/Pyramid")
+        tracked_request = TrackedRequest.instance()
+        span = tracked_request.start_span(operation="Controller/Pyramid")
 
+        try:
             if ignore_path(request.path):
-                tr.tag("ignore_transaction", True)
+                tracked_request.tag("ignore_transaction", True)
+
+            tracked_request.tag("path", request.path)
```

Comment:
Any reason we moved this outside the `try`? `.path` should supposedly always be present, but it's not critical in order for the TrackedRequest to work, and there's no harm in having it within the `try` where we do `user_ip` as well, right?

Reply:
`request.path` was already accessed outside of the `try` above - when I started writing a test to make it crash, I realized this.